portality/tasks/journal_in_out_doaj.py

```diff
@@ -70,20 +80,23 @@ def prepare(cls, username, **kwargs):
         job.user = username
         job.action = cls.__action__
 
-        journal_id = kwargs.get("journal_id")
+        journal_ids = kwargs.get("journal_ids")
 
-        params ={}
-        params["set_in_doaj__journal_id"] = journal_id
+        params = {}
+        params["set_in_doaj__journal_ids"] = journal_ids
         params["set_in_doaj__in_doaj"] = kwargs.get("in_doaj")
 
-        if params["set_in_doaj__journal_id"] is None or params["set_in_doaj__in_doaj"] is None:
+        if params["set_in_doaj__journal_ids"] is None or params["set_in_doaj__in_doaj"] is None:
```

Comment:
also, Bulk Edit classes have an example of validating incoming params and making some required, then you just call `_job_parameter_check` on this line. It encapsulates the param check so that prepare and run (which both need to check the params) use the exact same piece of code.